bin/setup

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env ruby
-require 'fileutils'
+require "fileutils"
 
 # path to your application root.
-APP_ROOT = File.expand_path('..', __dir__)
+APP_ROOT = File.expand_path("..", __dir__)
 
-DB = if ENV['DATABASE_URL'].to_s.empty?
-       'PracticalDeveloper_development'
+db_url = ENV["DATABASE_URL"].to_s
+
+# We explicitly unset DATABASE_URL for Gitpod so bin/setup can do its job.
+# See: https://github.com/thepracticaldev/dev.to/issues/7040
+DB = if db_url.to_s.empty? || db_url = "postgresql://gitpod@localhost"
```

Comment:
I tried **not** to do this, but doing it via `.gitpod.yml` and the `gp` command didn't work as expected, so after 3 tries I gave up.